src/parsers/transactions.ts

```diff
@@ -396,7 +396,7 @@ const parseTransaction = async (
           address:
             toLower(updatedAsset?.address) === ETH_ADDRESS
               ? ETH_ADDRESS
-              : toChecksumAddress(updatedAsset.address),
+              : toChecksumAddress(updatedAsset.address)!,
```

Comment:
It seems that since `updatedAsset.address` will be an address string, this should return a value successfully, but there might be a better way to handle this.

Reply:
Yeah, it's a tricky one. This is probably part of a bigger conversation around error handling that TypeScript is surfacing. Dealing with this might be beyond the scope of a straightforward migration exercise because it'll likely introduce more meaningful runtime code changes.

Reply:
hey for this one - we actually don't really need this `toChecksumAddress` check here for asset addresses; let's remove it and confirm tests pass